src/get-started/debug-and-profile.md

```diff
@@ -22,48 +23,49 @@ Build: $Id: ca5318932770ca063fc9885b4764c30bfaf8a199 $
 
 ### Attach delve to a running TiDB process
 
-As mentioned in [Get the code, build and run](build-tidb-from-source.md), once you get the TiDB server running, you can try attaching the delve debugger.
+Once you get the TiDB server running, you can try attaching the delve debugger.
 
 For example, you can build and run a standalone TiDB server by running the following commands in the root directory of the source code.
 
-```
-$ make server
-$ ./bin/tidb-server
+```bash
+make server
+./bin/tidb-server
 ```
 
 
```

Comment:
Please remove this blank line.

Reply:
Yes. I think we can integrate sth like markdown-lint later.